API_CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ file.
 
 This file will be updated weekly, if there are notable changes.
 
+## 2018-06-25
+
+## Checkout
+
+* Added `transfered` Payment State
+* Correctly add Line Items for deposits referenced by other line items
```

Comment:
Hier dann auch: `FIXED adding line items for deposits referenced by other line items`